debian/changelog

```diff
@@ -1,3 +1,9 @@
+sipauthserve-public (5.1.0) unstable; urgency=medium
```

Comment:
That's the distribution; I guess it defaults to 'unstable', one of the Debian branches. Not important for us.